packages/test/src/helpers.ts

```diff
@@ -38,7 +38,9 @@ export const REUSE_V8_CONTEXT = inWorkflowContext() || isSet(process.env.REUSE_V
 export const RUN_TIME_SKIPPING_TESTS =
   inWorkflowContext() || !(process.platform === 'linux' && process.arch === 'arm64');
 
-export const TESTS_CLI_VERSION = inWorkflowContext() ? '' : process.env.TESTS_CLI_VERSION;
+// TODO: Remove after next CLI release
+export const TESTS_CLI_VERSION = inWorkflowContext() ? '' : 'v1.3.1-priority.0';
```

Comment:
This should be done by just setting the env var in the GH workflows. See here: https://github.com/temporalio/sdk-typescript/pull/1679/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR22